### PR TITLE
Tool damage rebalance

### PR DESCRIPTION
**Pull request by Flareguy**
## About the PR
Rebalances most of the tools to no longer deal only 10 blunt damage.

The crowbar now does 6 blunt since it was really strong w/mob mentality and overshadowed most other crew weapons. It also deals 3 structural in addition however, so it's still good at destroying windows.
The red crowbar now deals two more blunt more than the base one.
The screwdriver and wrench are weaker so they don't outshine the crowbar, and should now probably have their own small niche.
The shovel now does 14 blunt damage because it's a fucking shovel.
Toolboxes deal 12 damage like they did in SS13.
Wirecutters and cow tools now deal pitiful amounts of damage.
Power tools can now deal damage.
...and some other more minor stuff not worth mentioning.

**Media**
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- tweak: Most tools no longer deal 10 blunt damage, and are across-the-board weaker.
- tweak: The red crowbar is now stronger than the grey crowbar.
- tweak: Shovels and toolboxes are now much more robust.
- tweak: Power tools can now deal damage.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Tools/welders.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@
     qualities: Welding
   - type: Welder
     litMeleeDamageBonus:
-      types: # When lit, negate standard melee damage and replace with heat
+      types:
         Heat: 10
         Blunt: -10
```
Comment (asperger-sind) [inline Resources/Prototypes/Entities/Objects/Tools/welders.yml:55]: Doesn't this just heal 5 blunt damage on-hit now

File: Resources/Prototypes/Entities/Objects/Tools/toolbox.yml [hunk 2]
```diff
@@ -11,12 +11,12 @@
   - type: Storage
     capacity: 60
   - type: Item
-    size: 9999
+    size: 110
```
Comment (ficcialfaint) [inline Resources/Prototypes/Entities/Objects/Tools/toolbox.yml:14]: Tests are failing cause of this. Syndie toolbox can contain items up to 170 weight, while its own weight is 110

**Review by ficcialfaint — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Tools/welders.yml (hunk 1 above)
Comment (Flareguy) [reply Resources/Prototypes/Entities/Objects/Tools/welders.yml]: whoops

File: Resources/Prototypes/Entities/Objects/Tools/toolbox.yml (hunk 2 above)
Comment (Flareguy) [reply Resources/Prototypes/Entities/Objects/Tools/toolbox.yml]: fixed

**Comment by deltanedas** [comment]
while youre at it give jaws some damage since ce only having welder as a weapon roundstart is a meme

**Comment by Flareguy** [comment]
> while youre at it give jaws some damage since ce only having welder as a weapon roundstart is a meme

fair enough. it is probably unlikely changing the jaw type would do anything meaningful (at least when compared to power drill screw head vs. wrench head) so i might as well

**Comment by freeman2651** [comment]
id personally rather not have the differently-colored-crowbar do more damage for really no reason but that's just me

**Comment by DawBla** [comment]
Shouldn't it be the other way around with emergency crowbar doing less damage?
People will just rob every emergency toolbox on the station and arrivals shuttle for imba crowbar

**Comment by Emisse** [comment]
sure

**Pull request merged**